<ui version="4.0" >
 <class>PortsWindow</class>
 <widget class="QWidget" name="PortsWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="tvPortList" >
      <property name="contextMenuPolicy" >
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
     </widget>
     <widget class="QStackedWidget" name="swDetail" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="portDetail" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbApply" >
            <property name="text" >
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <widget class="QFrame" name="frAggregate" >
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Capacity</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLineEdit" name="lineEdit_3" />
           </item>
           <item row="0" column="2" >
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Aggr fps</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3" >
            <widget class="QLineEdit" name="lineEdit" />
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>% age</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QLineEdit" name="lineEdit_4" />
           </item>
           <item row="1" column="2" >
            <widget class="QLabel" name="label_4" >
             <property name="text" >
              <string>Aggr bps</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3" >
            <widget class="QLineEdit" name="lineEdit_2" />
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QTabWidget" name="tabWidget" >
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3" >
           <attribute name="title" >
            <string>Control</string>
           </attribute>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <property name="leftMargin" >
             <number>0</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>0</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QTableView" name="tvStreamList" >
              <property name="contextMenuPolicy" >
               <enum>Qt::ActionsContextMenu</enum>
              </property>
              <property name="frameShape" >
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="lineWidth" >
               <number>0</number>
              </property>
              <property name="selectionMode" >
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <property name="selectionBehavior" >
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="portGroupDetail" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Select a port to configure streams</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="blankPage" />
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionNew_Port_Group" >
   <property name="icon" >
    <iconset resource="ostinato.qrc" >:/icons/portgroup_add.png</iconset>
   </property>
   <property name="text" >
    <string>New Port Group</string>
   </property>
  </action>
  <action name="actionDelete_Port_Group" >
   <property name="icon" >
    <iconset resource="ostinato.qrc" >:/icons/portgroup_delete.png</iconset>
   </property>
   <property name="text" >
    <string>Delete Port Group</string>
   </property>
  </action>
  <action name="actionConnect_Port_Group" >
   <property name="icon" >
    <iconset resource="ostinato.qrc" >:/icons/portgroup_connect.png</iconset>
   </property>
   <property name="text" >
    <string>Connect Port Group</string>
   </property>
  </action>
  <action name="actionDisconnect_Port_Group" >
   <property name="icon" >
    <iconset resource="ostinato.qrc" >:/icons/portgroup_disconnect.png</iconset>
   </property>
   <property name="text" >
    <string>Disconnect Port Group</string>
   </property>
  </action>
  <action name="actionNew_Stream" >
   <property name="icon" >
    <iconset resource="ostinato.qrc" >:/icons/stream_add.png</iconset>
   </property>
   <property name="text" >
    <string>New Stream</string>
   </property>
  </action>
  <action name="actionDelete_Stream" >
   <property name="icon" >
    <iconset resource="ostinato.qrc" >:/icons/stream_delete.png</iconset>
   </property>
   <property name="text" >
    <string>Delete Stream</string>
   </property>
  </action>
  <action name="actionEdit_Stream" >
   <property name="icon" >
    <iconset resource="ostinato.qrc" >:/icons/stream_edit.png</iconset>
   </property>
   <property name="text" >
    <string>Edit Stream</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ostinato.qrc" />
 </resources>
 <connections/>
</ui>
